lib/sensors/temperature.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <cstdint>
+#include <cstdio>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+static constexpr std::uint8_t kMux             = 0x70;
+static constexpr std::uint8_t kControl         = 0x04;
+static constexpr std::uint8_t kTempuratureHigh = 0x07;
+static constexpr std::uint8_t kTempuratureLow  = 0x06;
```

Comment:
I would prefer these to be of the form `k*insert reg name from datasheet*` with a little comment to the side to explain if there is any ambiguity. Eg, `kData_T_H  // high byte for the temperature value` so it can be very easily referenced with the datasheet. What do you think @kshxtij ?